Adjust cron times

keithbauer opened the pull request:
## Description

<!--- Describe your changes -->
Increase cron frequency for a few scripts. Ensures data integrity by running more often in case of transient failures. This is a summary of the call:

- axis reaper every 30 minutes to start with
- bibliotecha
    - purchase monitor once an hour
    - circulation sweep every 30 minutes
- Overdrive
    - new_titles minute 30 every fifth hour
    - overdrive_reaper
    - format_sweep every 30 minutes, reaper goes away because this should do the same job
- Enki
    - reaper often gives 500 errors, run this at least as often of the import, so maybe minute 30 of every 6th hour
- Spread out the OPDS for distributors a little bit
    - both should run once a day at different times of the day
- OPDS import from feedbooks should run once a day
    - think they moved from OPDS 1 to OPDS 2
- Next time we do work on the DPLA exchange we should look at the ODL import scripts
    - We’ll want these frequencies to be more like the overdrive and axis360 ones to run every 15 minutes, especially the `odl_import_monitor`
- Odilo doesn’t work, leave it alone
- ProQuest likely puts a ton of information into these OPDS feeds
    - `proquest_import_monitor` — maybe do once a day, but investigate
- SAML monitor is fine

## Motivation and Context

<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->
[SIMPLY-4085](https://jira.nypl.org/browse/SIMPLY-4085). The times were chosen on a call with Leonard on 2022-06-02.

## How Has This Been Tested?

<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->
I don't know how to test this. I double checked my times via https://crontab.guru/.

## Checklist:

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.

keithbauer committed:
Adjust cron times

Increase frequencies to offset failures for scripts that weren't running frequently.

mwbenowitz reviewed (approved):
These changes look great as far as the actual cron statements. A few questions about whats described in the PR:
- If feedbooks moved to OPDS2 do we need to update that script?
- I don't see an update for `odl_import_monitor` and `proquest_import_monitor`, should these be updated? Or do we need tickets to investigate what should be done
- What should we do for Odilo? Fix it or remove it would be the two options I think

I don't think these questions block merging this but I want to be sure we capture any questions and feature work somewhere!

keithbauer committed:
Move Odilo cron job to DEPRECATED section

keithbauer committed:
Run proquest_import_monitor daily

keithbauer committed:
Adjust ODL reapers to run more frequently

keithbauer commented:
Great questions!
* After some light investigation, I think feedbooks is still on OPDS1, although I do wonder about [this URL](https://github.com/NYPL-Simplified/circulation/blob/main/api/feedbooks.py#L69). I can't get this URL to resolve, but removing `/books/` from it resolves. I think this warrants further investigation.
*  I've adjusted ODL & Proquest to run more frequently.
* I've moved the Odilo script to the `DEPRECATED` section next to the RBdigital timings. I'm a bit hesitant to outright remove it, but if we think that's best I can do that.

mwbenowitz reviewed (approved):
LGTM!
